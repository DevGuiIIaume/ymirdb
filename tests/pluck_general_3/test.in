set a 1 2 3
type a
pluck a 2
type a
pluck a 2
pluck a 1
type a
set b 2
set c 2 a b
type b
type c
pluck c 2
type c
pluck c 1
type c
pluck c 1
type c
set c 1 a
pluck c 2
type c
bye
