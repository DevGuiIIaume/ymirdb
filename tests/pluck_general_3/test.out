> ok

> simple

> 2

> simple

> 3

> 1

> simple

> ok

> ok

> simple

> general

> a

> general

> 2

> general

> b

> simple

> ok

> a

> simple

> bye
